openpower/package/sb-signing-utils/sb-signing-utils.mk

```diff
@@ -0,0 +1,47 @@
+################################################################################
+#
+#  sb-signing-utils
+#
+################################################################################
+
+SB_SIGNING_UTILS_SITE = ssh://git@github.com/hellerda/sb-signing-utils.git
+SB_SIGNING_UTILS_SITE_METHOD = git
+SB_SIGNING_UTILS_SITE_VERSION = master
+
+SB_SIGNING_UTILS_LICENSE = Apache-2.0
+SB_SIGNING_UTILS_LICENSE_FILES = LICENSE
+SB_SIGNING_UTILS_VERSION ?= master
+
+HOST_SB_SIGNING_UTILS_DEPENDENCIES = host-openssl
+
+define HOST_SB_SIGNING_UTILS_PRE_CONFIGURE_CMDS
+	touch $(@D)/{NEWS,README,AUTHORS,ChangeLog}
```

Comment:
Yes, I am fixing that up now so the necessary files are there and the unnecessary ones (Changelog, NEWS) will be ignored.